### PR TITLE
K8SPSMDB-713 Rearrange backups doc and add physical backups

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

**Comment by egegunes** [comment]
https://percona-mongodb-operator-pr-37.onrender.com/backups.md is broken and it's accessible from welcome page

File: docs/backups-restore-to-new-cluster.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        * set `spec.backupSource` subsection instead of `spec.backupName` field
+            to point on the appropriate S3-compatible storage. This
+            `backupSource` subsection should contain a `destination` key,
+            followed by necessary storage configuration keys, same as in
+            `deploy/cr.yaml` file:
+
+            ```yaml
+            ...
+            backupSource:
+              destination: s3://S3-BUCKET-NAME/BACKUP-NAME
```
Comment (tplavcic) [inline docs/backups-restore-to-new-cluster.md:51]: type is missing

File: docs/backups-scheduled.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      type: s3
+      s3:
+        bucket: S3-BACKUP-BUCKET-NAME-HERE
+        region: us-west-2
+        credentialsSecret: my-cluster-name-backup-s3
+  tasks:
+   - name: "sat-night-backup"
+     enabled: true
+     schedule: "0 0 * * 6"
+     keep: 3
```
Comment (tplavcic) [inline docs/backups-scheduled.md:51]: add type here

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: docs/backups-move-from-external-db.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    s3:
+      region: us-west-2
+      bucket: pbm-test-bucket
+      credentials:
+        access-key-id: <your-access-key-id-here>
+        secret-access-key: <your-secret-key-here>
+    ```
+
+    After putting all needed details into the file (`AWS_ACCESS_KEY_ID`, 
+    `AWS_SECRET_ACCESS_KEY`, еру S3 bucket and region in the above example),
```
Comment (egegunes) [inline docs/backups-move-from-external-db.md:51]: what is epy?

File: docs/backups-move-from-external-db.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+        of the *source* and *destination* clusters, although Percona Backup for
+        MongoDB [allows replset-remapping](https://www.percona.com/blog/moving-mongodb-cluster-to-a-different-environment-with-percona-backup-for-mongodb/) as well.
+
+2. Configure the cloud storage for backups on your *source* cluster following
+    the [official guide](https://docs.percona.com/percona-backup-mongodb/initial-setup.html#configure-remote-backup-storage).
+    For example, using the Amazon S3 storage can be configured with the
+    following YAML file:
+
+    ``` yaml title="pbm_config.yaml"
+    type:s3
```
Comment (egegunes) [inline docs/backups-move-from-external-db.md:41]:
```suggestion
    type: s3
```

**Review by egegunes — CHANGES_REQUESTED** [review]

File: docs/backups-ondemand.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    ```yaml
+    apiVersion: psmdb.percona.com/v1
+    kind: PerconaServerMongoDBBackup
+    metadata:
+      finalizers:
+      - delete-backup
+      name: backup1
+    spec:
+      clusterName: my-cluster-name
+      storageName: s3-us-west
```
Comment (tplavcic) [inline docs/backups-ondemand.md:42]: I would specify type here as well. The more examples include it the more users will be aware that it exists.

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: docs/backups-ondemand.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+        (this name will be needed later to [restore the bakup](backups-restore.md)),
+
+    * `spec.clusterName` key should be set to the name of your cluster (prior to
+        the Operator version 1.12.0 this key was named `spec.psmdbCluster`),
+
+    * `spec.storageName` key should be set to the name of your [already configured storage](backups-storage.md).
+
+    * optionally you can set the `spec.type` key to `physical` if you would like
+       to make physical backups instead of logical ones (please see the
+       [physical backups limitations](backups.md#physical)). Otherwise set
```
Comment (tplavcic) [inline docs/backups-ondemand.md:27]: We seem to link to a file which is being deleted?

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: docs/backups-ondemand.md (hunk 6 above)
Comment (fiowro) [reply docs/backups-ondemand.md:27]: We restored this file several commits ago :)

File: docs/backups-ondemand.md (hunk 5 above)
Comment (fiowro) [reply docs/backups-ondemand.md:42]: done

File: docs/backups-restore-to-new-cluster.md (hunk 1 above)
Comment (fiowro) [reply docs/backups-restore-to-new-cluster.md:53]: fixed

File: docs/backups-scheduled.md (hunk 2 above)
Comment (fiowro) [reply docs/backups-scheduled.md:51]: done

**Pull request merged**